s3_file_field/views.py

```diff
@@ -78,13 +81,25 @@ def upload_initialize(request: Request) -> HttpResponseBase:
     upload_request: Dict = request_serializer.validated_data
     field = _registry.get_field(upload_request['field_id'])
 
+    file_name = upload_request['file_name']
     # TODO The first argument to generate_filename() is an instance of the model.
     # We do not and will never have an instance of the model during field upload.
     # Maybe we need a different generate method/upload_to with a different signature?
-    object_key = field.generate_filename(None, upload_request['file_name'])
+    object_key = field.generate_filename(None, file_name)
+
+    content_type = upload_request.get('content_type', None)
+    if content_type is None:
+        content_type = mimetypes.guess_type(file_name)[0]
```

Comment:
Guessing is OK, but if we want more deterministic / well defined behavior, we could just fall back to `application/octet-stream` as a generic type. If we do that, we can just do

```python
content_type = upload_request.get('content_type', 'application/octet-stream')
```

Reply:
Not specifying `Content-Type` in the metadata still results in S3 providing `Content-Type: application/octet-stream` when downloading it, so I would think just not specifying would be more sensible than explicitly recording it in metadata.

Reply:
Makes sense to me. If it's as simple as passing `ContentType=None` to the boto3 call, we can just do 

```python
content_type = upload_request.get('content_type')
```

Reply:
Is guessing the type a bad thing? I thought it was a useful sounding feature.

Reply:
It's not a bad thing. But I'm also not sure if it's a good thing, and if we are ambivalent, I would always err on the side of the simplest implementation.